webpack/ForemanTasks/Components/Chart/Chart.js

```diff
@@ -0,0 +1,128 @@
+/*
+  Copied from react-c3js: https://github.com/bcbcarl/react-c3js/blob/master/src/index.js
+  Added missing features:
+    * onChartCreate
+    * onChartUpdate
+
+  TODO: update react-c3js or move to patternfly-react
```

Comment:
It would be good to at least have the copied code in one commit and the additional patches you did to it in other, so that we see where we've diverged

Reply:
It would also help reviewing, so that we can focus on the new code that has been added